City-of-Pori/pori-kada
# PORI-51 current flow styling

To test:

1. build the site, drush updb -y && drush fra -y && drush cc all
2. Make sure you have all the types of current items (news item, liftup, blog post, twitter, facebook, instagram, youtube, linkedin) and that all of those are promoted to front page.
3. Go to https://local.pori.fi/admin/config/media/image-styles/edit/current and press "Override defaults" and save the image style.
4. Go to front page and make sure the styles on the block look good and responsive.

Layouts: 
https://drive.google.com/drive/u/0/folders/0B7ZHsug7PcEDVDBNTG51a0RIMzQ
https://drive.google.com/drive/u/0/folders/0B7ZHsug7PcEDVDBNTG51a0RIMzQ
